compute_tools/src/spec_apply.rs

```diff
@@ -703,9 +707,10 @@ async fn get_operations<'a>(
                                 )
                             } else {
                                 format!(
-                                    "CREATE ROLE {} {}",
+                                    "CREATE ROLE {} {} ROLE {}",
```

Comment:
this is (or at least should be) an equivalent of
```sql
CREATE ROLE foo;
GRANT foo TO neondb_owner;
```

ref https://www.postgresql.org/docs/current/sql-createrole.html
> ROLE role_name
>
>    The ROLE clause causes one or more specified existing roles to be automatically added as members, with the SET option enabled. This in effect makes the new role a “group”. Roles named in this clause with the role-level INHERIT attribute will have the INHERIT option enabled in the new membership. New memberships will have the ADMIN option disabled.